core/object.h

```diff
@@ -335,6 +335,11 @@ protected:
 			m_inherits::_notificationv(p_notification, p_reversed);                                                                     \
 	}                                                                                                                                   \
                                                                                                                                         \
+protected:                                                                                                                              \
+	virtual void _destruct() {                                                                                                \
```

Comment:
It seems like the spaces before the macro `\` is the issue...

C++ should really allow making multiline macros without having to escape every line 😆